### ISSUE TITLE
Migrating Travis to Github action

### ISSUE THREAD

**Issue by mtreacy002** [anitab-org/bridge-in-tech-backend#256]
## Description
As a maintainer,
I want to move the project CI from Travis CI to Github Action,
So that I can use the built-in Github CI and maintain consistency with projects across Anitab-org Open Source.

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
1 month

**Comment by devkapilbansal** [comment]
Seems like an easy issue. Should this be marked for OSH as well?

**Comment by vj-codes** [comment]
@mtreacy002 how's this different from #192 ?

**Comment by mtreacy002** [comment]
@vj-codes and @devkapilbansal , this is not an issue. It's an epic. This is to mark certain group of tasks in the project roadmap. If you open this pr on Google Chrome browser, you'll see the below screenshot inside this issue.
<img width="712" alt="Screen Shot 2021-04-06 at 7 55 41 pm" src="https://user-images.githubusercontent.com/29667122/113693704-3b98b080-9712-11eb-8747-283f4d4e43ca.png">

**Comment by mtreacy002** [comment]
If you look up Zenhub anitab-org/bridge-in-tech, you can see the project roadmap in the timeline with epics that contain their relevant tasks.
<img width="1342" alt="Screen Shot 2021-04-06 at 8 05 10 pm" src="https://user-images.githubusercontent.com/29667122/113694903-a5658a00-9713-11eb-8ed1-ffd0691ee11e.png">

**Comment by mtreacy002** [comment]
@isabelcosta , in the [BIT roadmap](https://github.com/orgs/anitab-org/projects/3) under the AnitaB.Org Open Source Projects, I can only see the project `Board` (below screenshot) and nothing else. I feel like the `Zenhub` version provides better options to view project's progress as can be seen in the screenshot above. It will be great if these analytical options be made available on the https://github.com/orgs/anitab-org/projects/ as well, @isabelcosta. Is this possible?

<img width="1406" alt="Screen Shot 2021-04-06 at 8 09 04 pm" src="https://user-images.githubusercontent.com/29667122/113697640-d1ced580-9716-11eb-8035-d02a1dbd1f02.png">

Also, @isabelcosta, I wonder who else can view the `anita-b/bridge-in-tech` zenhub workspace ([post above](https://github.com/anitab-org/bridge-in-tech-backend/issues/256#issuecomment-813999254)), it'll be good if everyone in OS leaders can have access to it too, as well as other project's workspace. @vj-codes or @devkapilbansal , can you please let me know whether or not you can view it?

**Comment by vj-codes** [comment]
@mtreacy002 um can you send a link? I know there was discussion about using zenhub and remember creating an account there but have no idea now , if there's a link I can confirm

**Comment by vj-codes** [comment]
@mtreacy002 I can view it on zenhub, just checked it although it's read-only for me mostly because I don't have write access to this repo and I think whoever's a part of the org can view it too

**Comment by epicadk** [comment]
Do I have to open it on chrome?

**Comment by mtreacy002** [comment]
> Do I have to open it on chrome? 

Yes, @epicadk . AFAIK Chrome works with this Zenhub plugin but Safari doesn't. I haven't tried it on other browsers 😉
